<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BoatTracker</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/icon.png</normaloff>images/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout">
    <property name="margin">
     <number>9</number>
    </property>
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="frameMap">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMap">
    <property name="title">
     <string>Map</string>
    </property>
    <addaction name="mpActionZoomIn"/>
    <addaction name="mpActionZoomOut"/>
    <addaction name="mpActionPan"/>
    <addaction name="mpActionAddLayer"/>
   </widget>
   <addaction name="menuMap"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="mpActionZoomIn">
   <property name="icon">
    <iconset>
     <normaloff>:/mActionZoomIn.png</normaloff>:/mActionZoomIn.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="mpActionZoomOut">
   <property name="icon">
    <iconset>
     <normaloff>:/mActionZoomOut.png</normaloff>:/mActionZoomOut.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="mpActionPan">
   <property name="icon">
    <iconset>
     <normaloff>:/mActionPan.png</normaloff>:/mActionPan.png</iconset>
   </property>
   <property name="text">
    <string>Pan</string>
   </property>
  </action>
  <action name="mpActionAddLayer">
   <property name="icon">
    <iconset>
     <normaloff>:/mActionAddLayer.png</normaloff>:/mActionAddLayer.png</iconset>
   </property>
   <property name="text">
    <string>Add Layer</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
